src/parser/mod.rs

```diff
@@ -3641,22 +3670,31 @@ impl<'a> Parser<'a> {
     #[must_use]
     pub fn parse_keyword(&mut self, expected: Keyword) -> bool {
         if self.peek_keyword(expected) {
-            self.next_token_ref();
+            self.advance_token();
```

Comment:
I think this is nicer too as it makes clear the token is unused (it is just being advanced)